datahub-web-react/src/app/entity/dataProduct/DataProductEntitiesTab.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useEntityData } from '../shared/EntityContext';
+import { EmbeddedListSearchSection } from '../shared/components/styled/search/EmbeddedListSearchSection';
+import generateUseListDataProductAssets from './generateUseListDataProductAssets';
+
+export function DataProductEntitiesTab() {
+    const { urn } = useEntityData();
+
+    return (
+        <EmbeddedListSearchSection
+            useGetSearchResults={generateUseListDataProductAssets({ urn })}
```

Comment:
Hmm.... Idk of this works because listDataProducts may not accept other filter types... Can we just reuse search with a fixedFilter? Or is that not possible?

Reply:
listDataProducts does accept other filter types! it takes in `SearchAcrossEntitiesInput` for queries and filters on the list of assets